### PR TITLE
bump gm version

### PR DESCRIPTION
**Pull request by richardmonette**
Updated for shopify/shopify https://github.com/Shopify/shopify/pull/95470

Brings dev into parity.

@mkcny @burke cc @Thibaut

### CODE REVIEW

**Comment by mkcny** [comment]
We'll need to update the bottled version as well, I think? Last time @burke got that ready and provided the hash for the bottle section.

**Comment by richardmonette** [comment]
cc @sbfaulkner

**Comment by richardmonette** [comment]
@burke good to 🚢 ?

**Comment by sbfaulkner** [comment]
@richardmonette hoping this will rbe produce that production problem? I'll look at more recent snapshots than this tonight to see if we can fix

**Comment by sbfaulkner** [comment]
(Reproduce)

**Comment by sbfaulkner** [comment]
@burke is there a simple way for me to test this version locally? eg. brew install from the downloaded `GraphicsMagick-1.4.020160531.tar.bz2` file or something?

File: shopify-graphicsmagick.rb [hunk 1]
```diff
@@ -5,9 +5,9 @@
 class ShopifyGraphicsmagick < Formula
   desc "Image processing tools collection"
   homepage "http://www.graphicsmagick.org/"
-  url "http://apt.shopify.com/dist/graphicsmagick/GraphicsMagick-1.4.020150705.tar.bz2"
-  sha256 "334fa009c7a1b5b91849b971ec85dffff19fb23737568cfdb49aee853731e7a5"
-  revision 2
+  url "http://apt.shopify.com/dist/graphicsmagick/GraphicsMagick-1.4.020160531.tar.bz2"
+  sha256 "ac0a46e99d4fc9e77f647a7e7e5bd3c80c600ec75441c52ebdb0a0966ad50ae9"
+  revision 3
```
Comment (sbfaulkner) [inline shopify-graphicsmagick.rb]: since we're bumping to `1.4.020160531` ... can we just go back to revision `1` ?

**Comment by sbfaulkner** [comment]
fwiw... I used `brew edit shopify-graphicsmagick` to apply these changes locally and then `brew unlink shopify-graphicsmagick` and `brew install shopify-graphicsmagick` (unsure if there's a better/easier way)

File: shopify-graphicsmagick.rb (hunk 1 above)
Comment (burke) [inline shopify-graphicsmagick.rb]: I really don't understand what's going on with our GM versions but someone tried to explain it to me one time and I didn't understand. It's crazy.

**Comment by burke** [comment]
I don't really have any tricks for testing this.

**Comment by sbfaulkner** [comment]
closing since this was never merged... it can be re-opened if still relevant /cc @haeky @tskorupa

**Pull request closed without merging**